scripts/testnet/include/config.sh

```diff
@@ -106,12 +107,30 @@ updateElasticsearchIndices() {
   popd
 }
 
+updateConfigFile() {
+  if [ "$ADDRESS_HRP" == "erd" ]; then
+    return 1
+  fi
+
+  file="$1"
+
+  grep -o '"erd1[^"]*"' "$file" | tr -d '"' | sort -u | while read -r addr; do
+    new_addr=$(python3 $MULTIVERSXTESTNETSCRIPTSDIR/convert_address.py $addr $ADDRESS_HRP)
+    escaped_old=$(printf '%s\n' "$addr" | sed 's/[\/&]/\\&/g')
+    escaped_new=$(printf '%s\n' "$new_addr" | sed 's/[\/&]/\\&/g')
+    sed -i "s/$escaped_old/$escaped_new/g" "$file"
+  done
+}
+
 copyNodeConfig() {
   pushd $TESTNETDIR
   cp $NODEDIR/config/api.toml ./node/config
   cp $NODEDIR/config/config.toml ./node/config/config_validator.toml
+  updateConfigFile ./node/config/config_validator.toml
```

Comment:
I think the safest way to do it here would be to iterate all over config folder.
Not a strong opinion, though